lib/service/storage/elasticsearch.ts

```diff
@@ -2918,7 +3116,8 @@ class ElasticSearch extends Service {
    * @throws If there is not exactly one indice associated
    */
   async _getIndice(index, collection) {
```

Comment:
All those private method should lost their underscore and gain the `private` modifier

Reply:
Done for most of them, BUT some methods exist both with and without underscore, so I do not rename them.